admin/src/app/components/ProducerTable.tsx

```diff
@@ -0,0 +1,143 @@
+"use client";
+
+import { useState } from "react";
+import { useRouter } from "next/navigation";
+import dayjs from "dayjs";
+
+import { patchFarms } from "@admin/_actions/farm/patch-farms";
+
+import { IProducer } from "@shared/interfaces/user";
```

Comment:
Interfaces com o prefixo I não serão mais utilizadas após o merge do PR #169. Será necessário atualizar quando ele for aprovado.